powerVCF.psm1

```diff
@@ -3001,6 +3001,78 @@ Function Get-VCFvRSLCMEnvironment {
 }
 Export-ModuleMember -Function Get-VCFvRSLCMEnvironment
 
+Function New-VCFvRSLCM {
+<#
+    .SYNOPSIS
+    Deploys vRealize Suite Lifecycle Manager
+    
+    .DESCRIPTION
+    Deploys vRealize Suite Lifecycle Manager to the specified network.
+    
+    .EXAMPLE
+    PS C:\> New-VCFvRSLCM -json .\SampleJson\vRealize\New-vRSLCM.json
+    This example Deploys vRealize Suite Lifecycle Manager using a supplied json file
+    
+    #>
+
+    param (
+        [Parameter (Mandatory=$true)]
+            [ValidateNotNullOrEmpty()]
+            [string]$json
+    )
+
+    if (!(Test-Path $json)) {
+        Throw "JSON File Not Found"
+    }
+    else {
+        try {
+            # Call Function createHeader to set Accept & Authorization
+            createHeader 
+            # Read the json file contents into the $ConfigJson variable
+            $ConfigJson = (Get-Content -Raw $json)
+            $uri = "https://$sddcManager/v1/vrslcms"
+            $response = Invoke-RestMethod -Method POST -URI $uri -headers $headers -ContentType application/json -body $ConfigJson
+            $response
+        }
+        catch {
+            # Call Function ResponseExecption to get error response from the exception
```

Comment:
Typo in comment ResponseException
To be fair I fixed this probably after you created your branch